UpdatePublishedVersions.ps1

```diff
@@ -16,12 +16,8 @@ param(
     # A pattern matching all packages in the set that the versions repository should be set to.
     [Parameter(Mandatory=$true)][string]$nupkgPath)
 
-& "$PSScriptRoot\dotnet.cmd" msbuild /nologo /verbosity:minimal /clp:Summary /nodeReuse:false `
-    /p:RestoreDefaultOptimizationDataPackage=false `
-    /p:PortableBuild=true `
-    /p:UsePartialNGENOptimization=false `
-    /maxcpucount `
-    "$PSScriptRoot\eng\updateversions.proj" /t:UpdatePublishedVersions `
+& "$PSScriptRoot\eng\common\build.ps1" -restore -build `
```

Comment:
Since this script doesn't have to do build tools initialization now, is there a reason for us to still have a separate script instead of just having the step in yml call build.ps1 directly?

Reply:
Should we also pass -ci? Considering this is mean only for use in the pipeline?